### PR TITLE
fixed: infer datatypes from sequence_by to __START_AT, __END_AT 

### PR DESCRIPTION
**Pull request by ravi-databricks**
fixed: infer datatypes from sequence_by to __START_AT, __END_AT for a…
removed: github actions for release
modified: setup.py for version

### CODE REVIEW

File: src/dataflow_pipeline.py [hunk 1]
```diff
@@ -253,16 +253,20 @@ def cdc_apply_changes(self):
             else self.silver_schema
         )
 
+        sequenced_by_data_type = None
+
```
Comment (alexott) [inline src/dataflow_pipeline.py:257]: where it's checked that `cdc_apply_changes.sequence_by` exists? if you don't have it, then `sequenced_by_data_type` won't be initialized...

Comment (ravi-databricks) [reply src/dataflow_pipeline.py:257]: its checking during onboarding process as mandatory field: [__validate_apply_changes](https://github.com/databrickslabs/dlt-meta/blob/issue/apply_changes_infer_version_col_datatype/src/onboard_dataflowspec.py#L510):

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 **NOTE:** For CLI interfaces, we support SemVer approach. However, for API components we don't use SemVer as of now. This may lead to instability when using dbx API methods directly.
 
 [Please read through the Keep a Changelog (~5min)](https://keepachangelog.com/en/1.0.0/).
+
+## [v0.0.21] - 2023-06-07
```
Comment (alexott) [inline CHANGELOG.md:12]: Changelog entry needs to be updated as well.

File: setup.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
     """
 setup(
     name="dlt_meta",
-    version="0.0.2",
+    version="0.0.2.1",
```
Comment (alexott) [inline setup.py:22]: Why not simply increase it to `0.0.3`?

**Review by alexott — APPROVED** [review]
lgtm. just minor comments

**Review by ganeshchand — APPROVED** [review]
We discussed the review comments. LGTM

**Pull request merged**